examples/helloworld/src/main/java/example/HelloWorld.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package example;
+
+import com.google.common.io.Resources;
```

Comment:
For examples like this it's good if we can keep dependencies to an absolute minimum, not even Guava.

Reply:
This example was intended to show how Jib can pack dependencies, resources, and classes into separate layers and that running the final image runs the application accordingly. I included Guava as an example dependency. Should I remove it and leave it as just a class and a resource instead?